### PR TITLE
Fix MSVC build

### PR DESCRIPTION
**Pull request by dnachbaur**

### CODE REVIEW

File: deflect/ImageWrapper.h [hunk 1]
```diff
@@ -67,7 +67,7 @@ enum CompressionPolicy {
  * (disabled by default).
  * @version 1.0
  */
-struct DEFLECT_API ImageWrapper
```
Comment (rdumusc) [inline deflect/ImageWrapper.h:70]: Ha! This was puzzling me too, I wasn't sure what it was doing here... :-)

File: apps/DesktopStreamer/CMakeLists.txt [hunk 2]
```diff
@@ -38,8 +38,8 @@ if(APPLE)
   else()
     list(APPEND DESKTOPSTREAMER_LINK_LIBRARIES "-framework CoreGraphics")
   endif()
-elseif(WIN32)
-  list(APPEND DESKTOPSTREAMER_LINK_LIBRARIES Ws2_32)
+elseif(MSVC)
+  list(APPEND DESKTOPSTREAMER_LINK_LIBRARIES Ws2_32 Qt5::WinMain)
```
Comment (rdumusc) [inline apps/DesktopStreamer/CMakeLists.txt]: I'm trying to see if this couldn't be done by common_gui_application() or common_qt_support(). From what I get it is only needed for GUI applications.

Comment (dnachbaur) [inline apps/DesktopStreamer/CMakeLists.txt]: Yes, it has a missing link to WinMain. Probably related to Qt5 port, so I would guess it's only needed for qt_support. Console applications have a regular main, so it's not needed there.

**Comment by dnachbaur** [comment]
What's missing for merge?

**Comment by rdumusc** [comment]
I was waiting for you to use https://github.com/Eyescale/CMake/pull/339

**Comment by dnachbaur** [comment]
Ok

**Comment by dnachbaur** [comment]
Updated

**Pull request merged**